hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -17,25 +17,20 @@ plugins {
 
 // Pinned to counter various CVEs with previous versions. Please remove this once Spring pulls in at least this version: https://docs.spring.io/spring-boot/docs/current/reference/html/dependency-versions.html
 ext["snakeyaml.version"] = "1.33"
-// Pinned to counter various CVEs. Please remove once Spring Boot brings in at least this version.
-ext["spring-security.version"] = "5.7.5"
-
-dependencyManagement {
-  imports {
-    mavenBom("software.amazon.awssdk:bom:2.18.28")
-  }
-}
 
 dependencies {
-  implementation("com.amazonaws:amazon-sqs-java-messaging-lib:2.0.1")
-  implementation("software.amazon.awssdk:sns")
+  implementation(platform("io.awspring.cloud:spring-cloud-aws-dependencies:3.0.0-M3"))
+  implementation(platform("software.amazon.awssdk:bom:2.18.28"))
```

Comment:
seems a nicer way to import a bom

Reply:
That's interesting. I only switched to `dependencyManagement { imports {...` very recently and it was to fix a problem when trying to publish the library. See this PR (I guess we'll find out when trying to publish locally) https://github.com/ministryofjustice/hmpps-spring-boot-sqs/pull/106

Reply:
In fact if you check the POM generated in your `build` library it will be red if you have this problem